src/nand.js

```diff
@@ -0,0 +1,28 @@
+import { and, complement } from 'ramda';
+
+/**
+ * Returns `true` if both arguments are falsy, otherwise `false`.
+ *
+ * @func nand
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.19.0|v2.19.0}
+ * @category Logic
+ * @sig a -> b -> Boolean
+ * @param {Boolean} a
+ * @param {Boolean} b
+ * @return {Boolean} True if both arguments are false.
+ * @example
+ *
+ * RA.nand(True, True); //=> False
```

Comment:
True, False are lowercase in `javascript`.

Reply:
Too much python recently I suppose